placementApp/choices.py

```diff
@@ -24,3 +24,12 @@
 YEAR_CHOICES = [("TE", "Third Year"), ("BE", "Fourth Year")]
 
 ROLE_CHOICES = [("STUDENT", "Student"), ("CO", "Co-ordinator"), ("TPO", "TPO")]
+
+CATEGORY_CHOICES = [("S", "Super Dream"), ("D", "Dream"), ("R", "Regular")]
+
+STATUS_CHOICES = [
+    ("1", "Schedule Interview"),
+    ("2", "Put Under Review"),
+    ("3", "Selected"),
```

Comment:
The order of status should be 1st put under review,  then either person is selected for an interview or rejected and if the person is selected for an interview then its either selected or rejected